paparazzi/src/test/java/app/cash/paparazzi/helper/TestSnapshotVerifier.kt

```diff
@@ -0,0 +1,36 @@
+package app.cash.paparazzi.helper
+
+import app.cash.paparazzi.Snapshot
+import app.cash.paparazzi.SnapshotHandler
+import app.cash.paparazzi.internal.ImageUtils
+import org.junit.rules.TemporaryFolder
+import java.awt.image.BufferedImage
+import java.io.File
+import javax.imageio.ImageIO
+
+internal class TestSnapshotVerifier(val tempDir: TemporaryFolder) : SnapshotHandler {
```

Comment:
```suggestion
internal class TestSnapshotVerifier(val failureDir: File) : SnapshotHandler {
```

nit: this class doesn't have to know about TemporaryFolder from an API perspective